src/riot-rs-embassy/src/network.rs

```diff
@@ -1,17 +1,30 @@
-//! To provide a custom network configuration, use the `riot_rs::config` attribute macro.
+//! Provides network access.
+//!
+//! The network link to use is selected through Cargo features.
```

Comment:
Should this also refer to the laze selection? Can a user of this crate expect enabling a backend through features to work?

Reply:
I was more thinking about the underlying laze mechanisms than the concrete features -- but as we don't have text on those in the book, I don't think there's something concrete to reference so far.